ckanext/adfs/extract.py

```diff
@@ -26,11 +26,9 @@ def get_user_info(saml):
             firstname = attr[0].text
         elif attr.attrib['Name'].endswith('surname'):
             surname = attr[0].text
-        elif attr.attrib['Name'].endswith('claims/name'):
```

Comment:
The claims/name should be a unique name for users. https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-R2-and-2012/ee913589(v=ws.11)#what-are-claim-types

However, we are using email addresses as usernames to prevent the public from registering with these names. The public will not be able to create accounts with email addresses as usernames.